src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -128,7 +132,15 @@ export class TechRecordSummaryComponent implements OnInit {
   }
 
   calculateVehicleModel(): void {
-    this.vehicleTechRecordCalculated = this.isEditing ? { ...cloneDeep(this.vehicleTechRecord), reasonForCreation: '' } : this.vehicleTechRecord;
+    this.isEditing
+      ? this.technicalRecordService.editableTechRecord$.pipe().subscribe(data => {
+          if (data) {
+            this.vehicleTechRecordCalculated = { ...cloneDeep(data), reasonForCreation: data.reasonForCreation ?? '' };
```

Comment:
```suggestion
            this.vehicleTechRecordCalculated = cloneDeep(data);
```
Surely the reason for creation of `data.reasonForCreation` is already `data.reasonForCreation`

Reply:
It starts as not existing on the tech record and is only added when you click amend, so on the first pass you need to create it which is the '', then every time you edit this is called again so it needs to place the same data back in.